# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.h
# Commit: d77d76d37d4d
# Full Hash: d77d76d37d4dccf39c12fa81c970088b5388d0f1
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-05-13 08:22:56
# Regressor Bug: 1518919
# File Overlap Count: 1
# Description:
#   Bug 1518919 - Propagate CSS visibility information to descendant documents via IPC call for fission. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30460
# ==============================================================================

diff -r 02b3eb8a1a13 -r d77d76d37d4d dom/base/nsFrameLoader.h
--- a/dom/base/nsFrameLoader.h	Thu May 09 07:37:03 2019 +0000
+++ b/dom/base/nsFrameLoader.h	Mon May 13 01:49:07 2019 +0000
@@ -128,6 +128,7 @@
   }
   nsresult CreateStaticClone(nsFrameLoader* aDest);
   nsresult UpdatePositionAndSize(nsSubDocumentFrame* aIFrame);
+  void SendIsUnderHiddenEmbedderElement(bool aIsUnderHiddenEmbedderElement);
 
   // WebIDL methods
 